.github/workflows/golangci-lint.yml

```diff
@@ -17,5 +17,5 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v2
         with:
-          version: v1.44.0
+          version: v1.45.0
```

Comment:
Just saw it, can you please bump to v1.45.2? (was just released yesterday)

Thank you very much!

(there are some go 1.8 related fixes in v1.45.1)